requirements.txt

```diff
@@ -117,7 +117,6 @@ rq==1.15.1
 rq-scheduler==0.13.1
 rubymarshal==1.0.3
 saneyaml==0.6.0
-scancode-toolkit==32.0.8
```

Comment:
Why did you unpin the SCTK version?

Reply:
Reverted back to pinning, there was a dependency conflict, fixed this the right way now. Thanks!